### PR TITLE
Updated a link that redirects to 404 to the meteor history link

### LINKED ISSUE #13321: link directs to 404 error in meteor update warning
The link in the update warning which appears in CLI returns 404.

`Meteor 3.0.2 is available. Check the changelog https://docs.meteor.com/changelog.html and update this project with 'meteor update'.`

The version of Meteor showing the problem: 
2.16

The expected behavior:
It should open https://v3-docs.meteor.com/history.html

The actual behavior.
https://docs.meteor.com/changelog.html gives 404.

### PR DESCRIPTION
**Pull request by mvorot**

### CODE REVIEW

**Comment by mvorot** [comment]
I realize this issue didn’t have a "ready" tag, but since it’s a very minor fix, I thought it would be a good opportunity for my first open-source contribution.

File: tools/packaging/updater.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ var maybeShowBanners = async function () {
     if (shouldShow(futureReleaseKey)) {
       runLog.log(
         "=> " + catalogUtils.displayRelease(track, futureReleases[0]) +
-        " is available. Check the changelog https://docs.meteor.com/changelog.html and update this project with 'meteor update'.");
+        " is available. Check the changelog https://v3-docs.meteor.com/history.html and update this project with 'meteor update'.");
```
Comment (leonardoventurini) [inline tools/packaging/updater.js]:
```suggestion
        " is available. Check the changelog https://docs.meteor.com/history.html and update this project with 'meteor update'.");
```

Now it's just `docs.`.

**Review by leonardoventurini — CHANGES_REQUESTED** [review]

**Review by denihs — APPROVED** [review]

**Comment by denihs** [comment]
Once the tests pass, we can merge it. Thank you!

**Comment by mvorot** [comment]
@denihs Build and Test fails because my forked brach is not found in upstream origin. Do you know how to fix it?

**Comment by denihs** [comment]
We are going to merge it like this. Thank you!

**Pull request merged**